lib/Api/cloud_services.dart

```diff
@@ -25,26 +31,35 @@ class CloudServices {
           },
         ),
       );
+
       if (response.statusCode == 200 || response.statusCode == 201) {
         final responseData = response.data;
         final token = responseData['token'];
         if (token == null || token.toString().isEmpty) {
           throw Exception('Invalid token received');
         }
         await storage.storeValue('auth-token', token);
-        final result = await storage.getValue('auth-token');
-        print('ofnjnj:$result');
+        showFlutterToast('Authenticated Successfully');
+        await Future.delayed(Duration(seconds: 1));
         found = true;
```

Comment:
nit: Add a delay of 1sec.
Adding a delay of 1sec to make the Ui look better. before adding this, it used to take user to home page and then show the toast as "Authenticated Successfully" which was not looking good. But after adding a delay it looks like a good flow.